library/src/main/java/com/getstream/sdk/chat/model/Channel.java

```diff
@@ -786,12 +790,24 @@ public void show(@NotNull CompletableCallback callback) {
         client.showChannel(this, callback);
     }
 
+    /**
+     * edit the channel's custom properties.
+     *
+     * @param options       the custom properties
+     * @param updateMessage message allowing you to show a system message in the Channel that something changed
```

Comment:
we should either have 2 methods (one with updateMessage and one without) or be more explicit that this is optional